src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(operand) {
+      this.result += operand;
+
+      return this;
+    },
+    subtract(operand) {
+      this.result -= operand;
+
+      return this;
+    },
+    multiply(operand) {
+      this.result *= operand;
+
+      return this;
+    },
+    divide(operand) {
+      this.result /= operand;
+
+      return this;
+    },
```

Comment:
You need to provide a check by 0, since you cannot divide by 0 and you need to throw an error
```suggestion
    divide(operand) {
      if (operand === 0) {
        throw new Error('Message error')
      } else {
        this.result /= operand;
      }

      return this;
    },
```